.storybook/story.css

```diff
@@ -0,0 +1 @@
+@import "tippy.js/dist/tippy.css";
```

Comment:
I don't think we need to do this to make tippy work in storybook, we already imported tippy.css with css loader in `.storybook/preview.js`. Also seems like this file is not imported to be used anywhere? It was an empty file before so I'm assuming it's safe to just delete it.

Reply:
You're right! I think we can get rid of this. I just put this in while I was trying to get things to work.